### PR TITLE
feat: add customBody prop to EntryCard

### PR DESCRIPTION
**Pull request by nekosaur**
# Purpose of PR

Adds `contentElement` prop to EntryCard. This is useful when you want to fully customize the look of an entry beyond the structured design allowed by `title`, `description` and `thumbnailElement` props.

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

**Comment by nekosaur** [comment]
Not sure if there's anything I can do about the failing checks? It seems to be a missing token?

**Review by gui-santos — CHANGES_REQUESTED** [review]
@nekosaur 

Thanks a lot for the contribution! It looks good
I have only one naming suggestion

I will take a look at the pipeline to check what's going on. Something is wrong with chromatic 😞

File: packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx
Comment (nekosaur) [reply packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx:85]: I'd be fine with that. I chose the name `contentElement` because of the existing `thumbnailElement` and `dropdownListElements` props.

**Comment by gui-santos** [comment]
@nekosaur for some reason the pipeline is not formating some CHANGELOG files and then it fails
I've just ran prettier on then and now you should be able to merge your PR 🙌

**Review by gui-santos — APPROVED** [review]
🚀

**Comment by nekosaur** [comment]
I can't merge it since it don't have write access.

**Comment by gui-santos** [comment]
> I can't merge it since it don't have write access.

@nekosaur I'm still trying to fix the chromatic problem 😭 

But as soon as I fix that I merge this and tell you
I think tomorrow we finally merge this

sorry for all these pipeline problems 🙏

File: packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx
Comment (suevalov) [inline packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx:85]: Looks like `customBody` acts like `children` here. Maybe it would be better to name it `children`?

**Review by suevalov — CHANGES_REQUESTED** [review]

File: packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx
Comment (nekosaur) [reply packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx:85]: Maybe? Naming it `children` maybe removes some semantic knowledge? I.e. if it is named `children` will someone think that it replaces everything inside the entry card? Similarly to how it works in a plain card?

**Comment by gui-santos** [comment]
I removed my approval because this PR is not getting attention anymore
And I realised that I think it's possible to achieve the result wanted by using the `Card` component

https://codesandbox.io/s/angry-bash-n7lip?file=/src/App.js

The Card component accepts `children` so it's very flexible

If this does not solve your demand @nekosaur , you can re-open the PR and re start the discussion 🙌

**Pull request closed without merging**
